CMakeLists.txt

```diff
@@ -158,26 +156,25 @@ if(GLAD_INSTALL)
   install(
     TARGETS glad
     EXPORT "${targets_export_name}"
-    LIBRARY DESTINATION "lib"
-    ARCHIVE DESTINATION "lib"
-    RUNTIME DESTINATION "bin"
-    INCLUDES DESTINATION "include")
-
-  if(GLAD_INCLUDE_DIRS)
```

Comment:
This check should still be necessary for the other languages?

Reply:
This should be language agnostic